notebooks/tm/osm/04-compare-osm-years.ipynb

```diff
@@ -0,0 +1,1512 @@
+{
```

Comment:
<p>Is this still needed? can delete if not used. but if relevant, run to show output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Ran to show output and added a markdown for clarity.</p>